mlflow/protos/mlflow_artifacts.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto2";
+
+package mlflow.artifacts;
+
+import "scalapb/scalapb.proto";
+import "databricks.proto";
+
+option java_package = "org.mlflow.api.proto";
+option py_generic_services = true;
+option (scalapb.options) = {
+  flat_package: true,
+};
+
+service MlflowArtifactsService {
+
+  rpc downloadArtifact (DownloadArtifact) returns (DownloadArtifact.Response) {
+    option (rpc) = {
+      endpoints: [{
+        method: "GET",
+        path: "/mlflow-artifacts/artifacts/<path:artifact_path>",
+        since { major: 2, minor: 0 },
+      }],
+      visibility: PUBLIC,
+      rpc_doc_title: "Download Artifact",
+    };
+  }
+
+  rpc uploadArtifact (UploadArtifact) returns (UploadArtifact.Response) {
+    option (rpc) = {
+      endpoints: [{
+        method: "POST",
```

Comment:
To comply with REST API conventions, I think this should be a `PUT` request instead of a `POST` request. See https://stackoverflow.com/questions/630453/what-is-the-difference-between-post-and-put-in-http#:~:text=POST%20means%20%22create%20new%22%20as,the%20server%20to%20create%20it.

Reply:
Got it, thanks for the knowledge!